packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -268,7 +268,9 @@ export class Combobox extends FormAssociatedCombobox {
 
         this.open = !this.open;
 
-        if (!this.open) {
+        if (this.open) {
+            this.control.focus();
+        } else {
             this.updateValue(true);
```

Comment:
You could move this to line 267 inside of the `if (this.open)` block. Also, when checking if that would cause a problem with how `this.updateValue()` works, I noticed that you only update the value if the controller is connnected. Should the emit be moved inside that block as well? Otherwise it will emit a change event sometimes when it's not changing.